CHANGELOG.md

```diff
@@ -90,6 +90,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed `allennlp.nn.util.add_sentence_boundary_token_ids()` to use `device` parameter of input tensor.
 - Be sure to close the TensorBoard writer even when training doesn't finish.
 - Fixed the docstring for `PyTorchSeq2VecWrapper`.
+- Made it possible to instantiate `TrainerCallback` from config files.
```

Comment:
Actually could you move this up to the "Unreleased" section?

Reply:
Ah whoops, fixed.